.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!--
+    If you are changing an existing module or resource, describe why that configuration is no longer acceptable. If you are fixing a misconfiguration, make sure to describe both the intended result and the actual (faulty) result.
+-->
+
+<!--
+    If you are adding a module or resources, describe what usage will be supported when this change is applied.
+-->
+
+<!--
+    Explain what alternative configurations you have evaluated and why the chosen configuration is preferable.
+-->
+
+<!--
+    What do reviewers need to know to review? Link pull requests or issues that are related.
+-->
+
+<!--
+    What do we need to consider when rolling out this change?
```

Comment:
I kind of like the approach of providing that information in the change log instead of having tho browse all PRs, e.g.  https://github.com/XenitAB/terraform-modules/releases/tag/2022.01.2

Reply:
You're right. I'll change it to a completeness checklist with docs, release notes, et c; the things the checks don't cover.

Reply:
@NissesSenap is of the opinion that we should not include details about breaking changes in the changelog itself. If that's the case, then it will have to be in the PR description, whence they can be copied to release notes. I don't have a strong opinion either way.